pkg/registry/core/seed/strategy.go

```diff
@@ -87,6 +87,8 @@ func mustIncreaseGeneration(oldSeed, newSeed *core.Seed) bool {
 		case v1beta1constants.GardenerOperationReconcile:
 			delete(newSeed.Annotations, v1beta1constants.GardenerOperation)
 			return true
+		case v1beta1constants.GardenerOperationRenewKubeconfig:
```

Comment:
Let's rather introduce a dedicated operation annotation for this - `renew-gardenlet-kubeconfig`. Otherwise, it can be very confusing which kubeconfig this is.

Reply:
The same annotation is used for ManagedSeed too. Shall we change it here too?